src/j1cli.ts

```diff
@@ -292,7 +291,7 @@ async function mutateEntities(j1Client, entities, operation) {
             );
             continue;
           }
-          entityIds = res.map(r => r.entity._id);
+          entityIds = res.map((r) => r.entity._id);
```

Comment:
eww what? I assume prettier did this, gross

Reply:
I'll do that rnrq

Reply:
I dislike it as well

Reply:
The reasoning behind this change is not purely stylistic. It's to improve editing experience for future developers. Personally, I think this was a good change, and I have been writing my code with parens for this reason all along. Here is the info:

https://prettier.io/blog/2020/03/21/2.0.0.html#change-default-value-for-arrowparens-to-always-7430httpsgithubcomprettierprettierpull7430-by-kachkaevhttpsgithubcomkachkaev

Reply:
Hmm. How often do you change an anonymous function to accept more arguments? Even if I did, adding parens around the parameters is four keystrokes for me. :thinking: